plugin/proto/algorithm_test.go

```diff
@@ -96,7 +97,7 @@ func TestHashAlgorithmFromKeySpec(t *testing.T) {
 	tests := []struct {
 		name     string
 		keySpec  signature.KeySpec
-		expected HashAlgorithm
+		expected plugin.HashAlgorithm
```

Comment:
should we keep `HashAlgorithm` for unit test, instead of `plugin.HashAlgorithm`?